x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_table/rules_table_saved_state.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FilterOptions, SortingOptions } from '../../../../rule_management/logic/types';
+
+export interface RulesTableSavedState {
+  tab?: string;
+  inMemory: boolean;
```

Comment:
This setting is already stored in local storage; no need to duplicate it. Also, per our discussion during the product meeting, we decided not to make it a part of the URL.